src/Form/Form.php

```diff
@@ -27,18 +27,10 @@ public function toJsonSchema(): array
     /**
      * Validate form values against the JSON Schema form
      * @param array<string, mixed> $values a key value array of form values
-     * @throws FormValidationException
+     * @throws ValidationException
      */
     public function validate(array $values): void
     {
-        /**
-         * CHECK_MODE_TYPE_CAST: Enable fuzzy type checking for associative arrays and objects
-         * src: https://github.com/jsonrainbow/json-schema?tab=readme-ov-file#configuration-options
-         **/
-        $validator = new Validator;
-        $validator->validate($values, $this->toJsonSchema(), Constraint::CHECK_MODE_TYPE_CAST);
-        if (!$validator->isValid()) {
-            throw new FormValidationException("Form validation failed", $validator->getErrors());
-        }
+        Validator::validate($values, $this->toJsonSchema());
```

Comment:
I would expect the result of the validator to be evaluated here and handle accordingly (throw exception if failed for example). See my [other comment](https://github.com/MyParcelCOM/json-schema-form-builder/pull/7/files#r2563867881) for more,